src/languageSupport/languages/flux/lsp/worker.ts

```diff
@@ -0,0 +1,31 @@
+// LSP worker code.
+//
+// IMPORTANT! All the code here will run in a web worker. This has ramifications in
+// the environment, e.g. there is no console, no dom, etc.
```

Comment:
no dom, but there is a console we can access. fyi.

Reply:
This seems to have iffy implementations across browsers, so I just noted it. It definitely works in chrome, but Firefox, even though it claims to support it.